packages/ipfs/docs/FAQ.md

```diff
@@ -88,8 +86,14 @@ const node = await IPFS.create({
   },
   libp2p: {
     modules: {
-      transport: [wstar],
-      peerDiscovery: [wstar.discovery]
+      transport: [WebRTCStar]
+    },
+    config: {
+      transport: {
+        WebRTCStar: {
```

Comment:
It'd be really nice if libp2p could make these keys case-insensitive.

Reply:
We will work on improving the config per [libp2p/js-libp2p#576](https://github.com/libp2p/js-libp2p/issues/576). It will probably change a lot the way we config libp2p. But referencing so that we have this in attention

Reply:
I just noticed in our config docs that we have this:

```js
const transportKey = WebRTCStar.prototype[Symbol.toStringTag]
```

Should we use it here to avoid confusions on where the naming comes from?

Reply:
Actually I'm not sure if that makes it any clearer.  It would be vastly preferable to be able to just use `[WebRTCStar.tag]` the same as the peer discovery config key.

Reply:
We should rething that in the config issue for libp2p. From what I know, the `tag` was added for the discovery side of things. The transports do not include it, except for webrtc-star, as it is both transport and discovery.
Anyway, we will look into it in a future libp2p release